### PR TITLE
system-tests and CI Visibility integration

### PR DESCRIPTION
**Pull request by robertomonteromiguel**
# What Does This Do
Integration system-tests and CI Visibility. Pipeline configuration to send data to CI Visibility

# Motivation
Observe the system-tests executions over main branch

# Additional Notes

### CODE REVIEW

**Comment by richardstartin** [comment]
We have a lot of forked processes in our smoke tests, will this integrate with them? I have a branch which can profile them but it's a bit artisanal.

File: .circleci/config.yml [hunk 1]
```diff
@@ -617,6 +619,14 @@ jobs:
             cd system-tests
             DD_API_KEY=$SYSTEM_TESTS_DD_API_KEY ./run.sh
 
+      - run:
+          name: Upload data to CI Visibility
+          command: |
+            cd system-tests
+            export DD_API_KEY=$DD_CI_API_KEY
+            export DD_APP_KEY=$DD_CI_APP_KEY
```
Comment (bantonsson) [inline .circleci/config.yml:627]: Please remove any global `DD_*` keys from circleci since it's failing all our cleanup checks in the tests.

**Review by bantonsson — CHANGES_REQUESTED** [review]
Fix this please https://app.circleci.com/jobs/github/DataDog/dd-trace-java/416545?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification

File: .circleci/config.yml (hunk 1 above)
Comment (robertomonteromiguel) [reply .circleci/config.yml]: Renamed. Now:

SYSTEM_TESTS_CI_API_KEY
SYSTEM_TESTS_CI_APP_KEY

**Review by bantonsson — APPROVED** [review]
Please squash the commits before merging to `master`

**Pull request merged**